spec/helpers/admin/claims_helper_spec.rb

```diff
@@ -114,4 +127,30 @@
       )
     end
   end
+
+  describe "#checking_deadline_warning" do
+    subject { helper.checking_deadline_warning(claim) }
+    before { travel_to Time.zone.local(2019, 10, 11, 7, 0, 0) }
```

Comment:
Is it important that it's a given date and not just frozen?

Reply:
Hmm yea, I guess the spec could be a little clearer. It needs to be frozen because it doesn't count weekends. I'll add another context